rollup.config.ts

```diff
@@ -34,7 +34,7 @@ const buildEsm = {
     'src/graphql.ts',
     'src/context/index.ts',
   ],
-  external: ['debug'],
+  external: ['debug', /@mswjs\/interceptors/],
```

Comment:
using a regex handles all relative imports from the package

Reply:
I wonder if we should mark all `@mswjs/` packages as external? They are all dependencies, so they should be installed alongside the `msw` package. The issue with updates propagation doesn't stop on `interceptors` alone, we've also got packages like `@mswjs/cookies`, for example.

On the other hand, listing such packages _explicitly_ may be a better idea. That'd save us from any unintended externalization of the dependencies if we ever decide to use a `@mswjs/` package as a dev dependency, for example.

What do you say if we extend this regular expression to include `cookies`?

```js
/@mswjs\/(cookies|interceptors)/
```